blog/2021-08-09-nft-minting-standard.md

```diff
@@ -31,12 +31,12 @@ To read more about CIP 721 and the NFT Metadata Standard, click [here](https://g
 
-Tokens that are built on Cardano are native. Unlike Ethereum’s ERC-20 tokens, Cardano doesn’t require a smart contract. Instead, the tokens are treated like Cardano’s native token, ADA. Using native tokens, you can create a variety of digital assets like Non-Fungible-Tokens representing art or appreciation. Learn more about this [here](https://developers.cardano.org/docs/native-tokens/).
+Tokens that are built on Cardano are native. Unlike Ethereum’s ERC-20 tokens, Cardano doesn’t require a smart contract. Instead, the tokens are treated like Cardano’s native token, ADA. Using native tokens, you can create a variety of digital assets like Non-Fungible-Tokens representing art or appreciation. Learn more about this [here](../docs/native-tokens/).
 
 To make sure the NFTAs came from the Cardano Foundation, a **policyId** identifies them. You can check out more information about the NFTAs and the **policyId** at the [Cardano Foundation NFTA Official Page](https://cardanofoundation.org/nfta). In addition, they are attached with [metadata](https://pool.pm/2783ee3048c5158646674def386e8610ce2c8824e515451baa4769a6.CFNFTA01) that describes the details of the transaction and ownership address.
 
 To mint the NFT’s, we first used a Cardano Foundation wallet to create a transaction to mint the tokens containing the metadata. Second, the ten developers sent us a receiving wallet address to receive the token. Third, another transaction to send the token to their wallet. Finally, to verify the work, each NFTA has a QR code that will navigate to the [Cardano Blockchain Explorer](https://explorer.cardano.org/en/address.html?address=addr1v8jad20nlga6ca7vjtqs2pr8cu0tcq3rq8s6svm546rvyjggl5yh7) for anyone to view and verify the transactions, including the Bronze Developer Portal Token issuance. Thus, no matter how much time goes by, the blockchain will have an immutable record to show that the community members received the first Non-Fungible Token of Appreciation.
 
 ![title image](/img/devblog/how-to-mint.png)
 
-At the time of writing this article, two NFTs have not been minted. We are waiting on two developers to provide their wallet addresses. Also, the Cardano Explorer is not showing the transactions correctly when you try to scan the QR code on the NFTAs. IOG is working on fixing this issue. We will keep you updated on when resolved. If you are interested in learning more or mint native assets or NFTs, visit the [Cardano Developer Portal](https://developers.cardano.org/docs/native-tokens/). 
+At the time of writing this article, two NFTs have not been minted. We are waiting on two developers to provide their wallet addresses. Also, the Cardano Explorer is not showing the transactions correctly when you try to scan the QR code on the NFTAs. IOG is working on fixing this issue. We will keep you updated on when resolved. Jump to [Discover Native Tokens](../docs/native-tokens/) page if you are interested in learning more or want to mint native assets or NFTs. 
```

Comment:
Can we please remove the sentence about the Cardano Explorer not showing the transaction. This was fixed.

Reply:
Fixed in PR #245